_layouts/blank.html

```diff
@@ -6,7 +6,7 @@
     {% include svg-icons.html %}
     <script>
       if ('serviceWorker' in navigator) {
```

Comment:
Since we are making changes, can me add a friendly console warning message if service workers are not supported?

Something like:
```js
if ('serviceWorker' in navigator) {
  // ...
} else {
  console.warn('Service workers are not supported by this browser. :(');
}
```